src/azul/terraform.py

```diff
@@ -708,6 +708,10 @@ def tf_config(self, app_name):
         for resource in resources['aws_lambda_function'].values():
             assert 'layers' not in resource
             resource['layers'] = ['${aws_lambda_layer_version.dependencies.arn}']
+            # Publishing the Lambda function as a new version prevents possible
+            # race conditions when an update to the function's configuration and
+            # code rely on the update of each other in order to work correctly.
```

Comment:
```suggestion
            # race conditions when there's a cyclic dependency between an update 
            # to the function's configuration and an update to its code
```

Reply:
I think my way is easier to read, assuming I've correctly parsed the intent here